docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:16.04
+
+# Update
+RUN apt-get update && apt-get dist-upgrade -y
+
+# Install the necessary packages.
+RUN apt-get install -y \
+    supervisor apache2 git php7.0 libapache2-mod-php7.0 php7.0-cli \
+    php7.0-mysqlnd php7.0-xml php7.0-mbstring php7.0-curl php7.0-zip
```

Comment:
2 things:
* you are correct that `php7.0-mysqlnd` is a mistake as there is no `php7.0-mysqlnd` package in Ubuntu 16.04. There was a "native" `php5-mysqlnd` package in Ubuntu 14.04 but for php 7 in ubuntu 16.04 it just uses the native driver anyways, and its now just called `php7.0-mysql`. The native driver added the support for things like async queries that php5-mysql didn't have.
* I'm no PDO expert (I prefer to just use pure mysqli), but I'm pretty sure that you still need to install the php7.0-mysql package in order to use PDO with a MySQL backend. This is because PDO is more like an application layer interface to make it so that you can program in a backend agnostic way. It [still needs the actual mysql package](http://polr.readthedocs.io/en/2.0-dev/user-guide/installation/) installed which it will make use of, if you want to use PDO with a MySQL backend. If one wanted to just use an sqlite database backend, then perhaps you wouldn't need this package, but I am guessing that a lot of users would want to tie it to a mysql database.

I tested just now with and without the php7.0-mysql package installed and only successfully ran the setup when I had added the php7.0-mysql package to the dockerfile. I have since pushed this change into this pull request. I don't know how this slipped through and can only guess that I added the nd stupidly after my initial testing.